src/packages/media/media/dropzone/dropzone.element.ts

```diff
@@ -1,17 +1,18 @@
-import { UmbDropzoneManager, type UmbUploadableFileModel } from './dropzone-manager.class.js';
+import type { UmbUploadableFolderModel, UmbUploadableFileModel } from './dropzone-manager.class.js';
+import { UmbDropzoneManager } from './dropzone-manager.class.js';
 import { UmbProgressEvent } from '@umbraco-cms/backoffice/event';
 import { css, html, customElement, property } from '@umbraco-cms/backoffice/external/lit';
-import type { UUIFileDropzoneElement, UUIFileDropzoneEvent } from '@umbraco-cms/backoffice/external/uui';
+import type { UUIFileDropzoneElement, UUIFileDropzoneEvent, UUIFileFolder } from '@umbraco-cms/backoffice/external/uui';
 import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
-import type { UmbTemporaryFileModel } from '@umbraco-cms/backoffice/temporary-file';
+import { UMB_NOTIFICATION_CONTEXT, type UmbNotificationContext } from '@umbraco-cms/backoffice/notification';
 
 @customElement('umb-dropzone')
 export class UmbDropzoneElement extends UmbLitElement {
 	@property({ attribute: false })
 	parentUnique: string | null = null;
 
 	@property({ type: Boolean })
-	multiple: boolean = true;
+	multiple: boolean = false;
```

Comment:
From my testing it is not possible to make a boolean property `false` if it's by default `true` therefore the default for `multiple` has to be `false` which also makes more sense when looking at the usage: `<umb-dropzone multiple></umb-dropzone>`

I've updated the current usages accordingly